### PR TITLE
Proposition for a restructured Getting Started guide

### PR DESCRIPTION
**Pull request by elric91**
Added simple example with step by step guide
Restructuration of the guide based on existing elements
Moved KMK vs CircuitPython in a dedicated page (which may need to be reviewed later to clarify KMK vs KMKPython)

English is not my native language so feel free to make it sound better if needed

### CODE REVIEW

**Comment by elric91** [comment]
Clarification on the difference between KMK and KMKPython (a dedicated guide on the specifics on KMKPython installation may be added in the future)

**Comment by elric91** [comment]
Added a correction to the name of the default file (code.py or main.py)

**Comment by elric91** [comment]
Added references to various existing documentation (which was not obvious to me)
Still missing a part to explain how to install KMKPython if we prefer it to CircuitPython

**Comment by elric91** [comment]
Keycodes doc enhancement (international keycodes as defined in keys.py)

File: docs/Getting_Started.md [hunk 1]
```diff
@@ -1,71 +1,94 @@
...
-extensions can do [here](extensions.md)
-
-## Debugging
-Debugging help can be found [here](debugging.md)
+> Life was like a box of chocolates. You never know what you're gonna get.
+
+KMK is a keyboard focused layer that sits on top of [CircuitPython](https://circuitpython.org/). As such, it should work with most [boards that support CircuitPython](https://circuitpython.org/downloads). It is best to use the last stable version (>5.0).
+Known working and recommended devices can be found [here](Officially_Supported_Microcontrollers.md)
+
+We are also providing a keyboard optimized version of CircuitPython (simplified to cope with memory limits of certain boards and with a selection of preinstalled relevant modules).If you're wondering why use KMKPython rather than barebone CircuitPython, we tried to compare both approaches [here](kmkpython_vs_circuitpython.md)
```
Comment (kdb424) [inline docs/Getting_Started.md:7]: Missing a space before "If you're wondering why use"

File: docs/Getting_Started.md [hunk 2]
```diff
@@ -1,71 +1,94 @@
...
+
+<br>
+
+## TL;DR Quick start guide
+> To infinity and beyond!
+1. [Install CircuitPython on your board](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython). With certain boards (pico, I look at you !), it can be as easy as drag and dropping the firmware on the drive
```
Comment (kdb424) [inline docs/Getting_Started.md:13]: Most boards are a simple drag and drop, so maybe omit the comment as it's not at all pico specific here.

File: docs/keycodes.md [hunk 3]
```diff
@@ -165,6 +165,31 @@
 |`KC.RIGHT_ANGLE_BRACKET`|`KC.RABK`, `KC.GT` |`>`                |
 |`KC.QUESTION`           |`KC.QUES`          |`?`                |
 
+## [International Keys]
```
Comment (kdb424) [inline docs/keycodes.md:168]: Thanks for adding these back. I stripped them as they were separated out into a module, but I guess they never actually left due to another PR adding them back, but without the docs.

**Comment by kdb424** [comment]
Added some comments, but generally it looks good. If you have no other thoughts, tag me in a comment when you are happy (once comments are resolved), and I'll merge. Thank you!

**Comment by elric91** [comment]
@kdb424 I updated my branch with your requests, hope it is the right way to do it ... should be OK

**Review by kdb424 — APPROVED** [review]

**Pull request merged**